### PR TITLE
Resizes /review table to fit viewport

### PR DESCRIPTION
**Pull request by coltonoscopy**
Makes necessary CSS and HTML changes to ensure /review table fits to the viewport 100% (with slight padding). Table resizes dynamically when window changes in size.

### CODE REVIEW

File: templates/review.html [hunk 1]
```diff
@@ -5,21 +5,21 @@
...
-                <th>Reviewed</th>
-                <th>id</th>
-                <th>cmd</th>
-                <th>timestamp</th>
-                <th>script</th>
+                <th width="7%">Reviewed</th>
+                <th width="5%">id</th>
+                <th width="15%">cmd</th>
+                <th width="10%">timestamp</th>
+                <th width="63%">script</th>
```
Comment (dmalan) [inline templates/review.html]: Deprecated in HTML5, I think. Better to use CSS selectors and have the template include the CSS file somehow!

File: templates/layout.html [hunk 2]
```diff
@@ -37,6 +38,17 @@
                 padding: 5px;
                 white-space: pre;
             }
+            
+            table {
+                width: 100%;
+                table-layout: fixed;
+                margin: 0;
```
Comment (dmalan) [inline templates/layout.html]: Let's alphabetize these and other properties for style's sake.

**Comment by coltonoscopy** [comment]
All set!

File: templates/review.html [hunk 3]
```diff
@@ -1,6 +1,11 @@
 {% extends "layout.html" %}
 {% block body %}
 {% if inputs | length > 0 %}
+<style>
+    .container {
+        width: 100%;
+    }
+</style>
```
Comment (dmalan) [inline templates/review.html]: Pretty sure HTML5 requires that this be in `head`. See https://github.com/cs50/help50/commit/688d3e11480bfcec0e35b0c76b33c6a155f6911c for a placeholder for you, whereby templating will put it where you want.

File: templates/review.html [hunk 4]
```diff
@@ -1,6 +1,11 @@
 {% extends "layout.html" %}
 {% block body %}
 {% if inputs | length > 0 %}
+<style>
+    .container {
+        width: 100%;
```
Comment (dmalan) [inline templates/review.html]: Is this necessary? Does `.container` not default to 100% in Bootstrap? Just curious.

File: templates/layout.html [hunk 5]
```diff
@@ -21,34 +21,66 @@
...
+                margin: auto;
                 position: relative;
+                top: 50%;
+            }
+
+            .th_reviewed {
+                width: 7%;
+            }
+
+            .th_id {
```
Comment (dmalan) [inline templates/layout.html]: I'd remove the `th_` prefix. Blurring the lines between metadata and markup. Plus, you're using these inside of `td`s too.

File: templates/layout.html [hunk 6]
```diff
@@ -21,34 +21,66 @@
...
+                width: 5%;
```
Comment (dmalan) [inline templates/layout.html]: Feels unnecessary to prescribe such specific percentages. Why not let Reviewed, id, and timestamp autosize based on their width, and then use `max-width` or such for `cmd` and `script`, which are the problematic ones?

File: templates/review.html (hunk 4 above)
Comment (coltonoscopy) [inline templates/review.html]: It doesn't; is something like 70 or 75%!

File: templates/review.html (hunk 3 above)
Comment (coltonoscopy) [inline templates/review.html]: Relocated to style tags!

File: templates/layout.html (hunk 5 above)
Comment (coltonoscopy) [inline templates/layout.html]: Removed!

File: templates/layout.html (hunk 6 above)
Comment (coltonoscopy) [inline templates/layout.html]: max-width doesn't fix (and would require either hardcoded pixels or percentages anyway). I'm not great at CSS, so I asked Jordan, who said we should be using Flex anyway for this sort of thing, but feels like too much to do a Flex refactor of the table. Do we need to worry about this detail, given this page will only be seen by very few folks anyway and the layout seems to work perfectly well as is? If you do think so, do you have other ideas in mind as to how to achieve? If not, I can ask Luke tomorrow what he thinks would be best.

**Comment by dmalan** [comment]
Tidied a bit! https://github.com/cs50/help50/commit/d35510c65922003bcffdcb7ebb91b209bdad0342

**Comment by coltonoscopy** [comment]
Looks great!

**Pull request merged**